### PR TITLE
feat: Navigation between created badges tab and management tab

### PR DESCRIPTION
**Pull request by Agupane**
Depends on #373 

# Description

- Routes the user from created badge tab to management tab if the user clicks a badge

### CODE REVIEW

File: src/pagePartials/profile/myProfile/ManageBadges.tsx [hunk 1]
```diff
@@ -39,6 +40,35 @@ export default function ManageBadges() {
...
+    {
+      title: t('badgeModelsList.filters.paused'),
+      color: 'blue',
+      defaultSelected: false,
+      key: BadgeModel_OrderBy.Paused,
+      filterType: 'Switch',
+    },
+    {
+      title: t('badgeModelsList.filters.thirdParty'),
+      color: 'error',
```
Comment (ramabit) [inline src/pagePartials/profile/myProfile/ManageBadges.tsx:53]: third-party has a green color in the home page, shouldn't we use the same one?

File: src/pagePartials/profile/myProfile/ManageBadges.tsx [hunk 2]
```diff
@@ -39,6 +40,35 @@ export default function ManageBadges() {
...
+      key: 'thirdParty',
+    },
+    {
+      title: t('badgeModelsList.filters.community'),
+      color: 'green',
```
Comment (ramabit) [inline src/pagePartials/profile/myProfile/ManageBadges.tsx:58]: community has pink color in the home page, shouldn't we use the same one?

File: src/pagePartials/profile/myProfile/ManageBadges.tsx [hunk 3]
```diff
@@ -39,6 +40,35 @@ export default function ManageBadges() {
...
+      key: 'kleros',
```
Comment (ramabit) [inline src/pagePartials/profile/myProfile/ManageBadges.tsx:59]: I guess in the long run we should change references to "kleros" for "community"

File: src/pagePartials/profile/myProfile/ManageBadges.tsx [hunk 4]
```diff
@@ -96,19 +161,27 @@ export default function ManageBadges() {
         minWidth={180}
         onViewPortEnter={() => {
           if (isMobile) {
-            setSelectedBadgeModelIndex(index)
+            router.replace({
+              pathname: router.pathname,
+              query: { ...router.query, modelId: bt.id },
+            })
```
Comment (ramabit) [inline src/pagePartials/profile/myProfile/ManageBadges.tsx:167]: please refactor function, this is duplicated

**Review by ramabit — CHANGES_REQUESTED** [review]

File: src/pagePartials/profile/myProfile/ManageBadges.tsx (hunk 3 above)
Comment (Agupane) [reply src/pagePartials/profile/myProfile/ManageBadges.tsx:59]: It's an internal key for the filter of the graph, on the UI it's displayed as community, to change the key we have to modify the subgraph, does not worth the effort now

File: src/pagePartials/profile/myProfile/ManageBadges.tsx (hunk 2 above)
Comment (Agupane) [reply src/pagePartials/profile/myProfile/ManageBadges.tsx:58]: Screenshot from 2023-12-29 16-26-02
done

File: src/pagePartials/profile/myProfile/ManageBadges.tsx (hunk 1 above)
Comment (Agupane) [reply src/pagePartials/profile/myProfile/ManageBadges.tsx:53]: done

File: src/pagePartials/profile/myProfile/ManageBadges.tsx (hunk 4 above)
Comment (Agupane) [reply src/pagePartials/profile/myProfile/ManageBadges.tsx]: done

**Review by ramabit — APPROVED** [review]

**Pull request merged**